PMSCore/Tester.cs

```diff
@@ -1,15 +1,18 @@
-﻿namespace PMSCore;
+﻿using System.Text.Json.Serialization;
+
+namespace PMSCore;
 
 /// <summary>
 /// Represents a Tester user, with permissions specific to tasks.
 /// </summary>
+[JsonConverter(typeof(UserJsonConverter))]
 public class Tester : User
 {
     /// <summary>
     /// Initializes a Tester with permissions to validate and report on tasks.
     /// </summary>
     public Tester(string userId)
-        : base(userId, new HashSet<Permission>
+        : base(UserType.Tester, userId, new HashSet<Permission>
         {
```

Comment:
: base(userId, new HashSet<Permission>
: base(UserType.Tester, userId, new HashSet<Permission>

I think the 1st one is redundant